pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[project]
+name = "clickhouse-sqlalchemy"
+dynamic = ["version"]
+description = "SQLAlchemy dialect for ClickHouse"
+authors = [
+    {name = "Konstantin Lebedev", email = "kostyan.lebedev@gmail.com"},
+]
+dependencies = ["alembic>=1.14.0", "asynch>=0.2.5", "clickhouse-driver>=0.2.9", "requests>=2.32.3", "sqlalchemy>=2.0.36"]
```

Comment:
Is alembic dependency really required by this package?

Reply:
Thank you, you are right, alembic got moved into the "test" dependency group.